programs/drift/src/state/user.rs

```diff
@@ -1281,6 +1281,16 @@ impl PerpPosition {
     pub fn is_bankrupt(&self) -> bool {
         self.position_flag & PositionFlag::Bankrupt as u8 > 0
     }
+
+    pub fn is_auto_transfer_to_cross_margin(&self) -> bool {
+        self.position_flag & PositionFlag::AutoTransferToCrossMargin as u8 > 0
+    }
+
+    pub fn can_transfer_isolated_position_deposit(&self) -> bool {
+        !self.is_open_position()
+            && !self.has_open_order()
```

Comment:
does this mean if user wanted to transfer but had an open order we would probably want to cancel that or show them an error saying you can't transfer because of this open order?

Reply:
yes